scripts/gWMS-CMSRunAnalysis.sh

```diff
@@ -87,11 +88,9 @@ echo "Hostname:   $(hostname -f)"
 echo "System:     $(uname -a)"
 echo "Arguments are $@"
 
+# dario: what does this do? do we need this?
```

Comment:
this line makes stderr to be redirected to stdout for all commands executed in the following part of the script. See e.g. https://stackoverflow.com/a/13088401
Whether is needed, I do not know on the spot.

Reply:
After thinking about this a bit more, I think we want this line because of how condor deals with command outputs. In DagmanCreator we have

```plaintext
Executable = gWMS-CMSRunAnalysis.sh
Output = job_out.$(CRAB_Id)
Error = job_err.$(CRAB_Id)
```

but in all CRABServer we have some logic about how to deal with `job_out.*`, not `job_err.*`. I think this `exec 2>&1` makes sure that `job_err.*` is empty.

Since it is pretty handy to have both stdout and stderr in the same file, so I would not change anything here! :)

thanks Stefano!

Reply:
I suggest adding a comment and/or a ptr to stackoverflow, or I will be confused again in 2 months !